src/modules/imageresizer/ImageResizerContextMenu/dllmain.cpp

```diff
@@ -227,7 +227,7 @@ class __declspec(uuid("8F491918-259F-451A-950F-8C3EBF4864AF")) ImageResizerConte
             auto val = get_last_error_message(GetLastError());
             Logger::warn(L"UuidCreate can not create guid. {}", val.has_value() ? val.value() : L"");
         }
-        else if (UuidToString(&temp_uuid, (RPC_WSTR*)&uuid_chars) != RPC_S_OK)
+        else if (UuidToString(&temp_uuid, static_cast<RPC_WSTR*>(static_cast<void*>( & uuid_chars))) != RPC_S_OK)
```

Comment:
2 static_cast isn't very readable, perhaps use reinterpret_/bit_ cast instead?

Reply:
Changed to reinterpret_cast.